client/src/components/Logo/Logo.test.js

```diff
@@ -1,16 +1,43 @@
-import React from 'react'
-import { shallow, configure } from 'enzyme'
-import Logo from './Logo.js'
-import Adapter from 'enzyme-adapter-react-16';
-configure({adapter: new Adapter()});
+import React from "react";
+import { shallow } from "enzyme";
+import Logo from "./Logo.js";
+import { findByTestAttribute } from "../../../../utils/utils";
 
-// We will describe a block of tests
-describe('Logo component', () => {
-	// we will write one individual test
-  it('should render as expected', () => {
-    // Shallow rendering renders a component without rendering any of its children
-    const component = shallow(<Logo />)
-    // We create an assertion within the test that checks if our component renders our name prop
-  // expect(component.contains('Your name')).toBe(true)
-  })
-})
+describe("Logo component", () => {
+  const setUp = props => {
+    const component = shallow(<Logo {...props} />);
+    return component;
+  };
+  it("Should render as expected", () => {
+    const wrapper = setUp();
+    const logo = findByTestAttribute(wrapper, ".Logo");
+    expect(logo.length).toBe(1);
+  });
+  it("Should render with small prop", () => {
+    const wrapper = setUp({ small: true });
+    const logo = findByTestAttribute(wrapper, ".Logo--small");
+    expect(logo.length).toBe(1);
+  });
+  it("Should render with clickable prop", () => {
+    const wrapper = setUp({ clickable: true });
+    const logo = findByTestAttribute(wrapper, "Link");
+    expect(logo.length).toBe(1);
+  });
+  it("Should render with clickable prop", () => {
+    const wrapper = setUp({ clickable: true });
+    const logo = findByTestAttribute(wrapper, "Link");
+    expect(logo.length).toBe(1);
+  });
```

Comment:
What exactly is this testing for? Wouldn't you want something like this?

`expect(wrapper.props().clickable).toBe(true)`?